tox.ini

```
@@ -6,6 +6,7 @@ description = run unit tests
 deps =
     pytest>=7
     -r requirements.txt
+    wheel>=0.38.4
```

Use the dev instead. This will also let you pass the lint error
```suggestion
    .[dev]
```

---

Thanks @tedhtchang, we'll see if the lint error passes now, but I think @alex-jw-brooks fixed this when he included `.[dev]` in the tox.ini file for his [PR 117](https://github.com/foundation-model-stack/fms-hf-tuning/pull/117)

---

Do you need 'wheel' to run pytest ?